readthedocs/config/tests/fixtures/for_schemaster/readthedocs.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "version": "enum('2')",
+  "formats": "any(list(enum('htmlzip', 'pdf', 'epub')), enum('all'), required=False)",
+  "conda": "include('conda', required=False)",
+  "build": "include('build', required=False)",
+  "python": "include('python', required=False)",
+  "sphinx": "include('sphinx', required=False)",
+  "mkdocs": "include('mkdocs', required=False)",
+  "submodules": "include('submodules', required=False)",
+  "redirects": "map(enum('page'), map(str(), str()), required=False)",
+  "python-install-requirements": {
+    "requirements": "path()"
+  },
+  "python-install": {
+    "path": "path()",
+    "method": "enum('pip', 'setuptools', required=False)",
+    "extra_requirements": "list(str(), required=False)"
+  },
+  "python-install-pipfile": {
+    "pipfile": "path()",
+    "dev": "bool(required=False)",
+    "ignore_pipfile": "bool(required=False)",
+    "skip_lock": "bool(required=False)"
+  }
+}
```

Comment:
This format isn't the one used by json schema, you should find the appropriate equivalents of the spec in yamale for json schema

Reply:
@stsewd i already have the yaml file should i upload the same to Schemaster , i would still be needing the test cases of the following , also what possible changes should i make in the given schema

Reply:
@KartikKapil you should upload the json schema here so we can review it.

Reply:
@stsewd alright thank you, will try to upload it asap.